Hours.java

```diff
@@ -1,13 +1,27 @@
 class Hours extends ConsoleProgram {
 
   /**
-  * Description
-  * @author:
+  * A program Hours.java that receives user input in hours and outputs
+  * its conversion in days and hours.
+  * @author: Kayla Jin
   */
   
   public void run() {
     
-    // start coding here
+    // Define variables
+    double dblHours;
+    double dblDays;
+    double dblDaysHours;
+
+    // Receive user input
+    dblHours = readDouble("Enter a number in hours: ");
+
+    // Perform calculations
+    dblDaysHours = dblHours % 24;
+    dblDays = (dblHours-dblDaysHours) / 24;
```

Comment:
Calculations are different from the solution to allow the conversion of hours to be in decimal form.